### PR TITLE
✨Source Customer.io: Migrate TS CDK to Low-code CDK

### LINKED ISSUE #29269: customer.io
https://docs.airbyte.com/integrations/sources/customer-io

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
*Migrating Source Customer.io to Low-Code CDK which is maintained by [Faros-ai](https://github.com/faros-ai/airbyte-connectors/tree/main/sources/customer-io-source)*
Closes #29269

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## Tests

<details><summary><strong>Integration & Acceptance</strong></summary>
Full Test Results

</details>

## 🚨 User Impact 🚨
- No breaking changes, migration to low-code
<details><summary><strong>Migrate Connector: Source Customer.io</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [x] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

</details>

### CODE REVIEW

**Comment by btkcodedev** [comment]
Test results, failing with incremental sync due to some date issue,
Full refresh is passing

image

**Comment by marcosmarxm** [comment]
Hello 👋 , right now Airbyte's integration tests are failing for this connector and I'm going to work during the week to fix them. Due a high number of contributions we have a backlog and your contribution will be review next week. We ask to you share the integration test from your side whatever possible and if you don't have credentials to run tests to get in touch with @RealChrisSean as soon as possible. Also some common missing steps contributors does migrating to low code are (and you can double check to make sure you implemented this): 
* adding paginator
* update the documentation changelog + metadata file and dockerfile version
* if the connector has `start_date` parameter you must implement incremental syncs

**Comment by btkcodedev** [comment]
Updated incremental sync and fixed,
 Ready for review :)

**Comment by btkcodedev** [comment]
All tests are passing in local

**Comment by btkcodedev** [comment]
creds: 
{
  "app_api_key": "f89c59e52e2a675420e97ae350816989",
  "start_date": "2023-08-17T00:00:00Z"
}

**Comment by btkcodedev** [comment]
Checks: 
- Incremental sync ✅ 
- Pagination: Not supported
- Version bumped ✅ 

All test passing.
Ping via slack for creds :)

**Review by marcosmarxm — APPROVED** [review]
Thanks @btkcodedev nice migration to official low code!

File: docs/integrations/sources/customer-io.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ Please follow the [their documentation for generating an App API Key](https://cu
 
 ## Changelog
 
-| Version | Date | Pull Request | Subject |
-| :--- | :--- | :--- | :--- |
-| 0.1.23 | 2021-11-09 | [126](https://github.com/faros-ai/airbyte-connectors/pull/126) | Add Customer.io source |
+| Version   | Date         | Pull Request                                                   | Subject                                       |
+| :-------- | :----------- | :------------------------------------------------------------- | :-------------------------------------------- |
+| 0.2.0     | 2021-11-09   | [29385](https://github.com/airbytehq/airbyte/pull/29385) | Migrate TS CDK to Low code                    |
```
Comment (rtol5) [inline docs/integrations/sources/customer-io.md:52]: Should this have a later date?

**Pull request merged**